docs/user-guide/cli-installcli.md

```diff
@@ -1,109 +1,116 @@
+:::note notes
 
--  To install Zowe CLI on a z/Linux operating system and you **require** the Secure Credential Store:
-   1. Follow the steps in [Configure Secure Credential Store on headless Linux operating systems](./cli-configure-scs-on-headless-linux-os.md).
-   2. Follow the steps in [Install Zowe CLI from npm](#install-zowe-cli-from-npm) or [Install Zowe CLI from a download](#install-zowe-cli-from-a-download).
--  To install Zowe CLI on a z/Linux operating system and you **do not require** the Secure Credential Store:
-   1. Follow the steps in [Install Zowe CLI from npm](#install-zowe-cli-from-npm) or [Install Zowe CLI from a download](#install-zowe-cli-from-a-download).
-   2. Follow the steps in [Configure Zowe CLI on operating systems where the Secure Credential Store is not available](./cli-configure-cli-on-os-where-scs-unavailable.md).
--  To install Zowe CLI on a USS system or on an operating system where you **cannot install** the Secure Credential Store:
-   1. Follow the steps in [Install Zowe CLI from npm](#install-zowe-cli-from-npm) or [Install Zowe CLI from a download](#install-zowe-cli-from-a-download).
-   2. Follow the steps in [Configure Zowe CLI on operating systems where the Secure Credential Store is not available](./cli-configure-cli-on-os-where-scs-unavailable.md).
+- As you install Zowe CLI, you might encounter **error messages** that relate to `cpu-features` and `ssh`. You can safely ignore error messages of this type; the installation completes successfully. This behavior can occur when you install CLI from npm and also from a local package.
 
-### Installation notes
+- **Linux users** might need to prepend `sudo` to `npm` commands. For more information, see [Known Zowe CLI issues](../troubleshoot/cli/known-cli.md#sudo-syntax-required-to-complete-some-installations).
 
-- As you are installing Zowe CLI, you might encounter error messages that relate to `cpu-features` and `ssh`. You can safely ignore error messages of this type; the installation completes successfully. This behavior can occur when you install CLI from npm and from a local package.
+:::
 
-## Prerequisites
+## Installing Zowe CLI and Zowe CLI plug-ins from a local package
 
-- Meet the [software requirements](../user-guide/systemrequirements-cli.md) for Zowe CLI.
-- Meet the [software requirements](../user-guide/cli-swreqplugins.md) for each plug-in.
+To install Zowe CLI from a local package:
 
-### Prerequisite notes
+1. Meet the [Zowe CLI software requirements](../user-guide/systemrequirements-cli.md).
 
-- If you are installing Zowe CLI on a computer that is running Node.js 16 on a Windows operating system, see [Installing Zowe CLI with Node.js 16 on Windows](../user-guide/cli-install-cli-nodejs-windows.md).
+2. Navigate to [Download Zowe](https://www.zowe.org/download.html). In the **Client-side component installer** section, click the **Zowe \<X.Y.Z\> CLI Core** button (where <X.Y.Z\> specifies the release number).
```

Comment:
The leading `<` is showing on the first X.Y.Z

Reply:
Thanks. The slash shows up in the local build but not on the live site. OK to leave as is.